Use Esperanto mnemonic seed for Esperanto GUI

ProkhorZ opened the pull request:
See issue #1131.

stoffu commented:
Looks like a lot of changes got reverted. Can you please squash commits?

moneromooo-monero commented:
The commits needs squashing. Looks like something like "git rebase HEAD~3" will work (and replace "pick" with "f" on the second and third lines).

ProkhorZ commented:
OK, wait a minute, please. I happen to have some tech support today.

ProkhorZ commented:
OK, I rebased with HEAD~3 as moo said, and I dropped the translation commit that got mixed up in here, as well as the reverting commit. Bit of a detour, sorry about that, but I learned to use git in a bash shell.
